packages/graphql/tests/performance/graphql/batch-create/batch-create-interface.graphql

```diff
@@ -1,4 +1,4 @@
-mutation BatchCreateInterface {
+mutation BatchCreateInterface_skip {
```

Comment:
This is meant to be skipped?

Reply:
Yes, this one was added after Simone changes, but it is really slow, making the perf. tests unbearably slow. Once we address that improvement we can remove the skip